compiler/runtime/runtime.c

```diff
@@ -23,13 +23,13 @@ SKIP_gc_type_t* get_gc_type(char* skip_object) {
 
 void SKIP_Regex_initialize() {}
 
-void SKIP_print_last_exception_stack_trace_and_exit() {
+void SKIP_print_last_exception_stack_trace_and_exit(void*) {
   todo();
 }
-void SKIP_unreachableMethodCall() {
+void SKIP_unreachableMethodCall(void*, void*) {
   todo();
 }
-void SKIP_unreachableWithExplanation() {
+void SKIP_unreachableWithExplanation(void*) {
```

Comment:
We should probably declare them `noreturn`

Reply:
They're technically not `noreturn` at the moment, as `todo()` throws a `RuntimeException` which can be caught.

Reply:
`noreturn` means the function will not finish its execution with `return` or reaching the end of the function, which is the case.

The exception can only be caught higher up in the stack, it doesn't prevent this function from being marked `noreturn`